akka-docs/src/test/scala/docs/actor/typed/BlockingDispatcherSample.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.actor.typed
+
+import akka.actor.typed._
+import akka.actor.typed.scaladsl._
+import com.typesafe.config.{ Config, ConfigFactory }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+// #blocking-in-actor
+object BlockingActor {
+  val behavior: Behavior[Int] = Behaviors.receiveMessage {
+    case i: Int =>
+      Thread.sleep(5000) //block for 5 seconds, representing blocking I/O, etc
+      println(s"Blocking operation finished: ${i}")
+      Behaviors.same
+  }
+}
+// #blocking-in-actor
+
+// #blocking-in-future
+object BlockingFutureActor {
+  def apply(): Behavior[Int] =
+    Behaviors.setup { context =>
+      implicit val executionContext: ExecutionContext = context.executionContext
+
+      Behaviors.receiveMessage {
+        case i: Int =>
+          println(s"Calling blocking Future: ${i}")
+          Future {
+            Thread.sleep(5000) //block for 5 seconds
+            println(s"Blocking future finished ${i}")
+          }
```

Comment:
break out this to a separate method "outside" the actor code, i.e. in `BlockingFutureActor` companion